modules/virt-deleting-custom-resource.adoc

```diff
@@ -4,15 +4,15 @@
 
 //This file contains UI elements and/or package names that need to be updated.
 
-[id="virt-deleting-kubevirt-hyperconverged-custom-resource_{context}"]
-= Deleting the CNV Operator Deployment custom resource
+[id="virt-deleting-custom-resource_{context}"]
```

Comment:
This file/id makes me think it would suit a generic module for removing any CR, whereas this is specifically removing the operator deployment CR. (We don't have a need for a generic module yet so we can happily cross that bridge if/when we come to it but raising it in case there's a better file/id)